kits/react-eslint/.eslintrc.cjs

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  root: true,
+  env: { browser: true, es2020: true },
+  extends: [
+    'eslint:recommended',
+    'plugin:react/recommended',
+    'plugin:react/jsx-runtime',
+    'plugin:react-hooks/recommended',
+  ],
+  ignorePatterns: ['dist', '.eslintrc.cjs'],
+  parserOptions: { ecmaVersion: 'latest', sourceType: 'module' },
+  settings: { react: { version: '18.2' } },
+  plugins: ['react-refresh'],
+  rules: {
+    'react-refresh/only-export-components': [
+      'warn',
+      { allowConstantExport: true },
+    ],
+  },
+}
```

Comment:
Chybí poslední prázdný řádek.

Reply:
@FilipChalupa Jak je to s tím prázdným řádkem na konci – musí tam být, nesmí, je lepší, když tam je nebo není? Mně připadá zbytečný… A už jsem se setkal s nástrojem, který i bílé znaky za poslední uzavírací závorkou JSONu hlásil jako chybu, protože přece tou závorkou je JSON ukončen a za ní už nic být nemá (považuju to tedy za chybu, ale musel jsem se otmu přizpůsobit).

Reply:
V VSCoce lze zapnou, ať se na konec souborů dává prázdný řádek automaticky. Stejně tak ho umí přidávat i Prettier. S problémem jsem se ještě nikdy nesetkal. Naopak volání například `cat soubor.txt` mi dělalo divnosti, když prázdný řádek na konci nebyl. Některé nástroje ho očekávají. Není náhoda, že GitHub na chybějící řádek upozorňuje červenou ikonkou.